pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
+extend-exclude = '^/examples/python/pytket-qujax_heisenberg_vqe.py'
```

Comment:
Why is this excluded? If it need changes I am happy to try to solve them.

Reply:
The issue is that we're using a jupyter-specific command to show the timing of a command (`%time`) :

https://github.com/CQCL/pytket/blob/f8fb88115a91045546be4570d821e96c2eae06b0/examples/python/pytket-qujax_heisenberg_vqe.py#L147 

https://github.com/CQCL/pytket/blob/f8fb88115a91045546be4570d821e96c2eae06b0/examples/python/pytket-qujax_heisenberg_vqe.py#L164

Since that is not python syntax, black errors out. We could, of course, just remove the `%time` command, but I was not sure how critical it was.

Reply:
We probably want to keep that, thank you for checking!

Reply:
Ideally we would remove the `%time` command from the python file (and use some valid python commands to measure the time), but that can be left for now.

Reply:
I have added that to the qujaxs tasks: https://github.com/CQCL/pytket-qujax/issues/85

Reply:
Just to make sure we don't forget it.